.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,59 @@
+name: Docker Build
+
+on:
+  push:
+    branches:
+      - "master"
+    tags:
+      - "*"
+  pull_request:
+
+env:
+  build_platforms: ${{ vars.BUILD_PLATFORMS || 'linux/amd64,linux/arm64/v8' }}
+  build_image: ${{ vars.BUILD_IMAGE || 'ghcr.io/isso-comments/isso' }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          flavor: |
+            latest=false
+          images: ${{ env.build_image }}
+          tags: |
+            type=ref,event=pr
+            type=semver,pattern={{major}}
```

Comment:
Where are these values (`{{major}}`, `{{version}}`) pulled from? `git` tags?

Reply:
That's correct, it's based on semver-formatted tags. More details here: https://github.com/docker/metadata-action#typesemver